text/0014-export.md

```diff
@@ -54,8 +55,11 @@ The components of a dataset use a standard naming scheme. Exporting a single com
 | header | "dataset.json" |
 | body | "body.json" |
 | transform script | "transform.sky" |
-| viz script | "viz.html" |    
-    
+| viz script | "viz.html" |
+| reference | "ref.txt" |
+
+The "dataset.json" file contains the header components that describe the dataset: the meta, structure, and commit. The "ref.txt" file contains the path of the dataset on the distributed web, as this value clearly can't be stored in the dataset itself, but it necessary for re-importing.
```

Comment:
The only thing I'd change here is changing "the path of the dataset on the distributed web" to "the full reference string, including human name and distributed web path"